stubs/php-file-get-contents/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run
+
+```console
+./run.php expired.badssl.com 443
+```
+
+# Dependencies
+
+ * php
```

Comment:
> Write a check in the code that returns UNSUPPORTED and prints the reeason if the default has been changed

My vote for this. Stub then works as an example for checking the required conditions instead of forcing something on which was turned off.